assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/index.test.js

```diff
@@ -0,0 +1,259 @@
+/**
+ * AudienceTilesWidget component tests.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import AudienceTilesWidget from '.';
+import { render } from '../../../../../../../../tests/js/test-utils';
+import {
+	createTestRegistry,
+	muteFetch,
+	provideModuleRegistrations,
+	provideModules,
+	unsubscribeFromAll,
+} from '../../../../../../../../tests/js/utils';
+import { getWidgetComponentProps } from '../../../../../../googlesitekit/widgets/util';
+import { availableAudiences } from '../../../../datastore/__fixtures__';
+import { MODULES_ANALYTICS_4 } from '../../../../datastore/constants';
+
+describe( 'AudienceTilesWidget', () => {
+	let registry;
+
+	const { Widget, WidgetNull } = getWidgetComponentProps(
+		'analyticsAudienceTiles'
+	);
+
+	const auduenceSettingsRegExp = new RegExp(
```

Comment:
I was in here while writing an IB for another ticket and noticed a typo here. auduence -> audience

Reply:
Thanks @benbowler ! I had noticed this and have included this in the CR feedback.